pkgs/development/libraries/gvfs/default.nix

```diff
@@ -82,6 +82,9 @@ stdenv.mkDerivation rec {
   doInstallCheck = doCheck;
 
   preFixup = ''
+    for f in $out/bin/*; do
+       substituteInPlace $f --replace 'exec $replacement' 'exec ${glib.dev}/bin/$replacement'
```

Comment:
How does replacing `$replacement` work?

Reply:
No magic: `$replacement` is just a shell variable reference inside the script, and it is always a shell command (starting with `gio`)

Reply:
@jtojnar see top of #48547